lib/helper.js

```diff
@@ -214,7 +214,7 @@ class Helper {
     let reject;
     const timeoutError = new TimeoutError(`waiting for ${taskName} failed: timeout ${timeout}ms exceeded`);
     const timeoutPromise = new Promise((resolve, x) => reject = x);
-    const timeoutTimer = setTimeout(() => reject(timeoutError), timeout);
+    const timeoutTimer = timeout ? setTimeout(() => reject(timeoutError), timeout) : 0;
```

Comment:
how did this come up? A quick glance says it will change the behavior of some public facing methods.

Reply:
The method has never been used before.